apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -98,9 +98,13 @@ $(document).ready(() => {
       cleanSchoolInfo();
       $('#user_age').val('21+');
     }
-    analyticsReporter.sendEvent(EVENTS.SIGN_UP_FINISHED_EVENT, {
-      'user type': user_type,
-    });
+    analyticsReporter.sendEvent(
```

Comment:
I think this is the event we thought should maybe move to the back end? Are we going to do that in a later PR? Or add that logging in addition to this?

Reply:
I was imagining adding both! But can remove this if it would be confusing

Reply:
Lets do both for now and we can remove it if it becomes confusing and the other is working better